### DISCUSSION TITLE
⏳ 🔗 ⛔ Locking header/footer edit in page editor

### DISCUSSION THREAD

**Discussion by mikewww19** [elementor/elementor#24184] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

When editing pages through Elementor, users will frequently click on the option to edit the header, footer or page templates like the top of a blogpost by accident when looking to alter part of the page. This leads to confusion and even frustration on their end, which we would like to prevent.

### Describe the solution you'd like

We would love to have a kind of "dummy mode" that can be enabled per customer, where we can restrict their access to these parts of the editor. The users would still have access through these in other parts of the website, but not through the Elementor editor

### Describe alternatives you've considered

We have attempted alternatives like whitelabeling and restricting to certain roles, but these end up being clunky and mostly non-functional

### Additional context

N.B.

**Comment by CDraines** [comment]
All we really need is one more option in the ROLE MANAGER  (PLEASE....;)
[ ] No access to editor
[ ] Access to edit content only
[X] CAN NOT EDIT TEMPLATES AND LOOPS

**Comment by chrisdarrell** [comment]
![roles](https://github.com/elementor/elementor/assets/48163951/a4e9e045-cef0-4147-b423-d219224e8942)

**Comment by Sergijus** [comment]
This is a great idea, but I'd love to see role manager broke down further, where we would be able to control who can do what with Elementor builder and templates :)

**Reply by chrisdarrell** [reply]
Agree, I'm for as much granularity as possible... But I'll start with the template thing...  Once the Quick edit was added to allow you to edit the header, footers, and loops.. I started having clients mess up header and loops.... :(

**Comment by TAGdevelop** [comment]
+1 +1 +1 !!!
@mikewww19 
Keep clients out of the headers/footers YES!

However our company uses templates for ads/specials that run every month - one template may be on 70% of pages throughout the site - the purpose is so someone can edit the single template and it changes on all the pages it is associated with.

So locking clients out of templates as a whole will create new problems for some clients that change their ads/promo's monthly. 

**Locking out of ALL templates is not perfect for me but certainly better than nothing.** There are more clients then not that should stay out of all templates.

I totally agree with more permission's control.

**Comment by githubtoh** [comment]
Just need to lock out template editing within the edit interface while in specific page/post editing mode. Should still allow in the backend and from the top black bar.
